generator/uscore/uscore_generator.rb

```diff
@@ -306,6 +309,71 @@ def create_search_test(sequence, search_param)
         )
       end
 
+      def create_chained_search_test(sequence, search_param)
+        # NOTE: This test is currently hard-coded because chained searches are
```

Comment:
could you raise an error in the generator if sequence not for PractionerRole just in case more are added in a future version?